packages/relay/src/lib/eth.ts

```diff
@@ -274,7 +280,8 @@ export class EthImpl implements Eth {
     this.cacheService = cacheService;
     this.mirrorNodeClient = mirrorNodeClient;
     this.precheck = new Precheck(mirrorNodeClient, logger, chain);
-    this.ethExecutionsCounter = this.initEthExecutionCounter(registry);
+    this.ethExecutionsCounter = this.initCounter('rpc_relay_eth_executions', ['method', 'function'], registry);
```

Comment:
Scratch `rpc_relay_daily_active_users` and let's reuse `rpc_relay_eth_executions` in the following form
```suggestion
    this.ethExecutionsCounter = this.initCounter('rpc_relay_eth_executions', ['method', 'function', 'from', 'to'], registry);
```

The rest of the changes seem good expect I think we should be able to add a test case to confirm we're capturing it